### DISCUSSION TITLE
Possibility of flatpak delta updates,

### DISCUSSION THREAD

**Discussion by ossenthi** [Jeffser/Alpaca#350] [Ideas]
I been using Alpaca for two or three months. Since then I have noticed update size for Flatpak getting large, about 1.6GB every time there is a change and update are quite frequent (which is a good thing!). I haven't explored change-log every time but the listed changes sometimes seems miner according to the description. I know Flatpak support delta updates, which means that only the updated data is downloaded during updates. Is it technically possible to utilize that in this project to reduce update size? 

PS:
I personally have no problem with speed or bandwidth wise but every miner improvement matter since some people have slow or limited internet access.

**Comment by Jeffser** [comment]
Sometimes the Ollama instance gets updated with support for new models, unfortunately I can't do delta updates of Ollama, that probably explains the multiple gig updates, Alpaca itself does delta updates tho

**Reply by ossenthi** [reply]
I figured, since Ollama alone is about 1GB. It be nice if we can officially  can access Alpaca's Ollama for other things like it mentioned in
https://github.com/Jeffser/Alpaca/discussions/334 since this is kept up to date more frequently. So no need for multiple Ollama instances in single machine and at lest we can save some bandwidth that way.

**Reply by Jeffser** [reply]
when running Alpaca, the Ollama instance is exposed to http://127.0.0.1:{port} where port is the number set inside the config file `~/.var/app/com.jeffser.Alpaca/config/server.json` which is `11435` by default
